### PR TITLE
Starred Contacts

### PR DESCRIPTION
**Pull request by vrrao01**

### CODE REVIEW

File: lib/pages/contact/contact.dart [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
+  "N",
+  "O",
+  "P",
+  "Q",
+  "R",
+  "S",
+  "T",
+  "U",
+  "V"
+];
```
Comment (vrrao01) [inline lib/pages/contact/contact.dart:35]: @Hareesh-Nandigrama What about the missing alphabets in this list? Ideally, we want an alphabet header to appear only for the contacts that exist, but we can't hardcode the alphabets for the dummy data. Try finding a way to make the appropriate headers appear irrespective of data.

File: lib/stores/contact_store.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  @action
+  void setStarredContacts(List<ContactDetailsModel> l) {
+    starredContacts = ObservableList<ContactDetailsModel>.of(l);
+  }
+
+  @computed
+  List<Widget> get starContactScroll {
+    if (starredContacts.length == 0) {
+      return [Text("You have no starred contacts")];
```
Comment (vrrao01) [inline lib/stores/contact_store.dart:35]: @Hareesh-Nandigrama  Can you add text style to this? Currently looks like: 
image

File: lib/pages/contact/contact.dart [hunk 3]
```diff
@@ -34,189 +55,223 @@ class _ContactPageState extends State<ContactPage> {
...
-                        ],
+        body: Provider<ContactStore>(
+          create: (context) => ContactStore(),
+          builder: (context, _) {
+            return Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(8, 14, 8, 14),
+                  child: ContactSearchBar(),
```
Comment (vrrao01) [inline lib/pages/contact/contact.dart:76]: @Hareesh-Nandigrama Current this searches across `ContactModel`s. I guess searching `ContactDetailsModel`s will be more useful. Or maybe combine both if that's feasible?

File: lib/stores/contact_store.dart (hunk 2 above)
Comment (Hareesh-Nandigrama) [reply lib/stores/contact_store.dart:35]: Done

File: lib/pages/contact/contact.dart (hunk 1 above)
Comment (Hareesh-Nandigrama) [reply lib/pages/contact/contact.dart]: Done

**Pull request merged**